### PR TITLE
Updated React Native SDK example

### PR DESCRIPTION
**Pull request by EmilianoSanchez**

### CODE REVIEW

File: ExpoExample/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  * [npm run android](#npm-run-android)
+  * [npm run eject](#npm-run-eject)
+
+## Setup
+
+You'll need [NodeJS](https://nodejs.org/en/download/). We recommend that you use the latest LTS version.
+
+Second thing you'll need is to install [Expo-CLI](https://expo.io/) with the command `npm install -g expo-cli`.
+
+To run the app, first change the `'<API-KEY>'` string in the App.js file with the browser key of your Split environment. Optionally, you can try a localhost configuration as the example below:
```
Comment (NicoZelaya) [inline ExpoExample/README.md:27]: , first **_replace_** the `'<API-KEY>'` string in the App.js file with .....

File: ExpoExample/__tests__/App.test.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+/**
+ * If you're looking for how to test with Split SDK, you can mock the module import (see https://jestjs.io/docs/jest-object#jestmockmodulename-factory-options)
+ * using the localhost (offline) mode of the SDK (see https://help.split.io/hc/en-us/articles/4406066357901-React-Native-SDK#localhost-mode)
+ * which is what we're doing here.
+ *
+ * It is not recommended to use the default (online) mode of the SDK in your tests because that slows down your tests and increases their flakiness due to network latencies.
+ * But if you want to do it anyway, be aware that you need to polyfill the 'fetch' API, which is used by the SDK but not provided by Jest (see https://www.npmjs.com/package/isomorphic-fetch).
```
Comment (NicoZelaya) [inline ExpoExample/__tests__/App.test.js:7]: Remove this line. We want to avoid this.

File: README.md [hunk 3]
```diff
@@ -1,128 +1,35 @@
...
+* Android [Github](https://github.com/splitio/android-client) [Docs](https://help.split.io/hc/en-us/articles/360020343291-Android-SDK)
+* GO [Github](https://github.com/splitio/go-client) [Docs](https://help.split.io/hc/en-us/articles/360020093652-Go-SDK)
+* iOS [Github](https://github.com/splitio/ios-client) [Docs](https://help.split.io/hc/en-us/articles/360020401491-iOS-SDK)
 * Java [Github](https://github.com/splitio/java-client) [Docs](https://help.split.io/hc/en-us/articles/360020405151-Java-SDK)
 * Javascript [Github](https://github.com/splitio/javascript-client) [Docs](https://help.split.io/hc/en-us/articles/360020448791-JavaScript-SDK)
 * Node [Github](https://github.com/splitio/javascript-client) [Docs](https://help.split.io/hc/en-us/articles/360020564931-Node-js-SDK)
-* .NET [Github](https://github.com/splitio/.net-core-client) [Docs](https://help.split.io/hc/en-us/articles/360020240172--NET-SDK)
-* Ruby [Github](https://github.com/splitio/ruby-client) [Docs](https://help.split.io/hc/en-us/articles/360020673251-Ruby-SDK)
+* Javascript for Browser [Github](https://github.com/splitio/javascript-browser-client) [Docs](https://help.split.io/hc/en-us/articles/360058730852)
```
Comment (NicoZelaya) [inline README.md:23]: This line should be placed after the regular `Javascript` one for alphabetical order.

File: ReactNativeExample/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
+# Split React Native SDK example application.
+
+This project was bootstrapped with [React Native CLI](https://reactnative.dev/docs/getting-started) .
+
+```sh
```
Comment (NicoZelaya) [inline ReactNativeExample/README.md:5]: It feels as if the script is missing a title or something

File: ReactNativeExample/README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  * [npm run ios](#npm-run-ios)
+  * [npm run android](#npm-run-android)
+
+## Setup
+
+You'll need [NodeJS](https://nodejs.org/en/download/). We recommend that you use the latest LTS version.
+
+Second thing you'll need is to install Android Studio and Xcode for Android and iOS development respectively. See [React Native - Setting up the development environment](https://reactnative.dev/docs/environment-setup) for more details.
+
+To run the app, first change the `'<API-KEY>'` string in the App.js file with the browser key of your Split environment. Optionally, you can try a localhost configuration as the example below:
```
Comment (NicoZelaya) [inline ReactNativeExample/README.md:25]: use replace here as well

File: ReactNativeExample/README.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+### `npm start`
+
+Starts the Metro bundler server that communicates with connected devices.
+
+### `npm test`
+
+Runs the [jest](https://github.com/facebook/jest) test runner on your tests.
+
+If you're looking for how to test with Split SDK, you can mock the module import (see [Jest documentation](https://jestjs.io/docs/jest-object#jestmockmodulename-factory-options)) using the [localhost mode](https://help.split.io/hc/en-us/articles/4406066357901-React-Native-SDK#localhost-mode) of the SDK, which is what we're doing [here](./__tests__/App.test.js).
```
Comment (NicoZelaya) [inline ReactNativeExample/README.md:67]: Note: Let's mention in the docs if not already there.

File: ReactNativeExample/__tests__/App.test.js [hunk 7]
```diff
@@ -0,0 +1,65 @@
+/**
+ * If you're looking for how to test with Split SDK, you can mock the module import (see https://jestjs.io/docs/jest-object#jestmockmodulename-factory-options)
+ * using the localhost (offline) mode of the SDK (see https://help.split.io/hc/en-us/articles/4406066357901-React-Native-SDK#localhost-mode)
+ * which is what we're doing here.
+ *
+ * It is not recommended to use the default (online) mode of the SDK in your tests because that slows down your tests and increases their flakiness due to network latencies.
+ * But if you want to do it anyway, be aware that you need to polyfill the 'fetch' API, which is used by the SDK but not provided by Jest (see https://www.npmjs.com/package/isomorphic-fetch).
```
Comment (NicoZelaya) [inline ReactNativeExample/__tests__/App.test.js:7]: remove line 7

File: ReactNativeExample/app.json [hunk 8]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "ReactNativeExample",
+  "displayName": "ReactNativeExample"
```
Comment (NicoZelaya) [inline ReactNativeExample/app.json:3]: Split React Native Example?

File: ReactNativeExample/README.md (hunk 6 above)
Comment (EmilianoSanchez) [reply ReactNativeExample/README.md:67]: Great!

File: ReactNativeExample/app.json (hunk 8 above)
Comment (EmilianoSanchez) [reply ReactNativeExample/app.json:3]: :/ changing the app name requires additional changes in native code. I will leave it for a future iteration.

**Pull request merged**